src/Core/MySQL/MySQLReplication.h

```diff
@@ -336,6 +336,10 @@ namespace MySQLReplication
             : EventBase(std::move(header_)), binlog_version(0), create_timestamp(0), event_header_length(0)
         {
         }
+        FormatDescriptionEvent(EventHeader & header_)
```

Comment:
Can we remove the ctor in l. 335?

(same for the corresponding moving ctors below)

Reply:
> Omitting std::move will throw error , because EventBase has only move ctor. I can add copy ctor in EventBase and remove std::move from 340 and so on . What do you suggest?

Yes, I was actually suggesting to add a copy ctor to EventBase + to remove EventBase's move ctor. This will necessitate removing all the std::move()-s, e.g in l. 340.

Reply:
I am also having some issues with CI today. Will try to start the tests manually.

Reply:
> I am also having some issues with CI today. Will try to start the tests manually.

Thanks @rschu1ze.

Reply:
@rschu1ze ,BuilderbinTidy is green and  I see failed intergration tests are timed out and seems unrelated to this PR.  Are we good to merge this PR?